README.md

```diff
@@ -1,93 +1,114 @@
+[Super JSON](https://zed.brimdata.io/docs/formats/zson) where all JSON values 
+are also Super JSON values.  Similarly,
+the [Super Binary](https://zed.brimdata.io/docs/formats/zson) format is an efficient
+binary representation of Super JSON (a bit like Avro) and the
+[Super Columnar](https://zed.brimdata.io/docs/formats/zson) format is a columnar
+representation of Super JSON (a bit like Parquet).
+
+Even though SuperDB is based on these super-structured data formats, it can read and write
+most common data formats.
+
+## Try It
+
+Trying out SuperDB is super easy: just [install](https://zed.brimdata.io/docs/#getting-started)
+the command-line tool [`super`](https://zed.brimdata.io/docs/commands/zq/).
+
+Detailed documentation for the entire SuperDB system and its piped SQL syntax
+is available on the [SuperDB docs site](https://zed.brimdata.io/docs).
+
+The SuperDB query engine can run locally without a storage engine by accessing
+files, HTTP endpoints, or S3 paths using the `super` command. While
+[earlier in its development](https://zed.brimdata.io/docs/commands/zed/#status),
+SuperDB can also run on a
+[super-structured data lake](https://zed.brimdata.io/docs/commands/zed/#1-the-lake-model)
+using the `suber db` sub-commands.
+
+## Piped Query Syntax
+
+The long-term goal for SuperDB's SQL syntax (SuperSQL) is to be Postgres-compatibe and interoperate 
+with BI tools though this is currently a roadmap item.  At the same time, the project
+seeks to forge now ground on the usability of SQL for data exploration.  To this end,
+SuperSQL supports the
+[pipe query syntax](https://github.com/google/zetasql/blob/master/docs/pipe-syntax.md)
+of GoogleSQL, recently described in their
+[VLDB 2024 paper](https://research.google/pubs/sql-has-problems-we-can-fix-them-pipe-syntax-in-sql/).
+
+In addition to the GoogleSQL syntax, SuperSQL includes additional pipeline 
+operators to enhance usuability, e.g., for search, for traversing 
+highly nested JSON, for data shaping, etc.
+
+To facilitate real-time, data exploration use cases,
+SuperDB supports an abbreviated form of SuperSQL called
+[SuperPipe](https://zed.brimdata.io/docs/language).
+
+SuperPipe provides a large number of shortcuts when typing interactive 
+queries, e.g., implied group-by clauses, dropping keywords,
+implied keyword searches, and so forth.  Even though SuperPipe is simply
+a short-hand form SuperSQL, it sort of looks like the pipeline-style
+languages utilized in search systems.
+
+### SuperDB Desktop - Coming Soon
+
+[SuperDB Desktop](https://github.com/brimdata/zui) is an Electron-based
+desktop app to explore, query, and shape data in SuperDB data lake.
+It combines a search experience with a SQL query and has some really slick
+design for dealing with complex and large JSON data.
+
+Unlike most JSON browsing tools, it won't slow to a crawl --- or worse crash ---
+if you load it up with ginormous JSON values.
```

Comment:
```suggestion
if you load it up with enormous JSON values.
```

I see "ginormous" _is_ a word in some dictionaries but my spell checker flagged it as if it's not, so maybe worth changing to something slightly more common. No offense taken if you really love "ginormous" though! 😄